discovery_engine_core/web-api/Cargo.toml

```
@@ -3,36 +3,38 @@ name = "web-api"
 version = { workspace = true }
 edition = { workspace = true }
 rust-version = { workspace = true }
+description = "Wep Api server for the document personalization service."
 license = "AGPL-3.0-only"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bincode = { workspace = true }
-bytes = { version = "1.2.1", features = ["serde"] }
-chrono = { workspace = true, features = ["serde"] }
-derive_more = { workspace = true }
+actix-web = { version = "4.2.1", default-features = false, features = ["compress-gzip", "compress-zstd"] }
+anyhow = { workspace = true }
+clap = { workspace = true }
+derive_more = { workspace = true, features = ["display", "deref", "as_ref", "into"] }
```

🔧 the first three features are already included in the workspace `Cargo.toml`, so they don't have to be repeated here.
```suggestion
derive_more = { workspace = true, features = ["into"] }
```
we could also add the last feature to the workspace `Cargo.toml` as it might be used in other places as well. then we don't have to repeat it here.
```suggestion
derive_more = { workspace = true }
```

---

I will add this to 

- #691